app/src/main/java/com/myskng/virtualchemlight/activity/MainActivity.kt

```diff
@@ -21,17 +20,19 @@ class MainActivity : AppCompatActivity() {
         binding = DataBindingUtil.setContentView(this, R.layout.activity_main)
         binding.handlers = this
         // Prepare ViewPager
-        viewPagerAdapter.PresenterList.add(ViewPagerPresenter(findViewById(R.id.tutorial_viewpager_layout_1)))
-        viewPagerAdapter.PresenterList.add(ViewPagerPresenter(findViewById(R.id.tutorial_viewpager_layout_2)))
-        viewPagerAdapter.PresenterList.add(ViewPagerPresenter(findViewById(R.id.tutorial_viewpager_layout_3)))
+        viewPagerAdapter.presenterList.add(ViewPagerPresenter(findViewById(R.id.tutorial_viewpager_layout_1)))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/5573df7eced397f9df793e114603442757174972/app/src/main/java/com/myskng/virtualchemlight/activity/MainActivity.kt#L23"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>